Gemfile

```diff
@@ -4,11 +4,3 @@ source "https://rubygems.org"
 ruby "2.4.1"
 
 gemspec path: "."
-gemspec path: "decidim-core"
-gemspec path: "decidim-system"
-gemspec path: "decidim-admin"
-gemspec path: "decidim-dev"
-gemspec path: "decidim-api"
-gemspec path: "decidim-pages"
-gemspec path: "decidim-comments"
-gemspec path: "decidim-meetings"
```

Comment:
Seems like the build failures are related, the development dependencies are missing. You're going to need to leave `decidim-dev` in there.